### PR TITLE
Refine remote debugging guide

### PR DESCRIPTION
**Pull request by st0012**
And also fix some issues found in readme.

### CODE REVIEW

**Comment by st0012** [comment]
@peterzhu2118 I'm preparing a proposal for the upstream so I need this to be merged now. If you find anything I'll still address them in a separate PR.

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ But a debugging session usually consists of 4 steps:
 
 > **Note**
 > You can connect the debugger to your program remotely through UNIX socket or TCP/IP.
-> To learn more, please check the [remote debugging guide](docs/remote_debugging.md).
+> To learn more, please check the [remote debugging guide](/docs/remote_debugging.md).
```
Comment (peterzhu2118) [inline README.md:52]: This link is broken for me.

File: docs/remote_debugging.md [hunk 2]
```diff
@@ -76,6 +79,22 @@ DEBUGGER__.open
 # your application code
 ```
 
+## UNIX Domain Socket (UDS)
+
+By default, the debugger uses UNIX doamin socket for remote connection. You can customize the socket directory and socket path with environment variables:
```
Comment (peterzhu2118) [inline docs/remote_debugging.md:84]: Typo:

```suggestion
By default, the debugger uses UNIX domain socket for remote connection. You can customize the socket directory and socket path with environment variables:
```

**Review by peterzhu2118 — COMMENTED** [review]
Sorry, I forgot to review this.

**Pull request merged**